app/views/pages/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="banner">
+  <div class="banner-info">
+    <h1>
+      <%= link_to "create a unique experience", <new_experience_path%></new_experience_path%>
+    </h1>
+    <p>
+    </p>
+  </div>
+</div>
+
+<div class="flexbox">
+  dropbown x2
+</div>
+
+<div class="container">
+
+  <div class="row">
+    <div class="card-experience">
+      <img src=''>
+    experiences.each do|e| do
```

Comment:
you're doing an each on "e" columns but you're calling the variable experience, you need to change e to experience

Reply:
ok